src/sections/Gsoc/index.js

```diff
@@ -0,0 +1,192 @@
+    <ul>
+      <li className="gsoc">
+        <b>Description</b>: Ensure that a service mesh is properly configured
+        and that its behavior conforms to official SMI specifications.
+        Conformance consists of both capabilities and compliance status as
+        outlined in the
+        <a
+          href="https://docs.google.com/document/d/1HL8Sk7NSLLj-9PRqoHYVIGyU6fZxUQFotrxbmfFtjwc/edit"
+          > design specification</a>
+      </li>
+      <li className="gsoc"><b>Recommended Skills</b>: Golang, Kubernetes</li>
+      <li className="gsoc">
+        <b>Mentor(s)</b>: Lee Calcote (<a href="https://twitter.com/lcalcote"
+          >@lcalcote</a
+        >), Sagar Utekar (<a href="https://twitter.com/named_uttu"
+          >@named_uttu</a
+        >)
+      </li>
+      <li className="gsoc">
+        <b>Issue(s)</b>:
+        <a href="https://github.com/servicemeshinterface/smi-spec/issues/70 "
+          > https://github.com/<wbr/>servicemeshinterface/<wbr/>smi-spec/issues/70
+        </a>
+      </li>
+      <li className="gsoc">
+      <b>Mentee:</b> 
+Naveen Jain (<a href="https://github.com/nveenjain">@nveenjain</a>)
+</li>
+    </ul>
+  </div>
+  <div>
+    <b className="blue">Distributed Load Testing of Service Meshes</b>
+    <ul>
+      <li className="gsoc">
+        <b>Description</b>: Many performance benchmarks are limited to single
+        instance load generation (single pod load generator). This limits the
+        amount of traffic that can be generated to output of the single machine
+        that the benchmark tool runs on in or out of a cluster. Overcoming this
+        limitation would allow for more flexible and robust testing. Distributed
+        load testing in parallel poses a challenge when merging results without
+        losing the precision we need to gain insight into the high tail
+        percentiles. Use Meshery’s interface for supporting different load
+        generators (a point of extensibility) to place support for Vegeta.
+        Workloads running on service meshes are hit in various ways and
+        generally not from the same source. Use of a distributed load generator
+        will aid in the testing of service mesh behavior.
+      </li>
+      <li className="gsoc"><b>Recommended Skills</b>: Golang, Kubernetes</li>
+      <li className="gsoc">
+        <b>Mentor(s)</b>: Prateek Sahu (<a
+          href="https://twitter.com/prateeksahu22"
+          >@PrateekSahu22</a>), Kanishkar J (<a href="https://twitter.com/_kanishkarj_"
+          >@_kanishkarj_</a>)
+      </li>
+      <li className="gsoc">
+        <b>Issue(s)</b>:
+        <a href="https://github.com/envoyproxy/envoy-perf/issues/72"
+          > https://github.com/<wbr/>envoyproxy/<wbr/>envoy-perf/issues/72
+        </a>
+      </li>
+            <li className="gsoc">
+      <b>Mentee:</b> 
+Kush Trivedi (<a href="https://github.com/kushthedude">@kushthedude</a>)
+</li>
+    </ul>
+  </div>
+  
+  
```

Comment:
Remove these unnecessary blank lines. From anywhere possible in this file.

Reply:
okay